### PR TITLE
Update Docs

### PR DESCRIPTION
**Pull request by harikrishnana2021**
Included PR welcome badge by replacing build badge

**What kind of change does this PR introduce?**
Included feature change 

**Did you add tests for your changes?**
yes

**Does this PR introduce a breaking change?**
yes

**What needs to be documented once your changes are merged?**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,7 +9,9 @@
 
 [![node][node]][node-url]
 [![deps][deps]][deps-url]
-[![tests][tests]][tests-url]
+<a href="https://gatsbyjs.org/contributing/how-to-contribute/">
+    <img src="https://img.shields.io/badge/PRs-welcome-brightgreen.svg" alt="PRs welcome!" />
+  </a>
```
Comment (evilebottnawi) [inline README.md]: Why? `gatsbyjs` !== webpack

File: README.md [hunk 2]
```diff
@@ -9,7 +9,7 @@
 
 [![node][node]][node-url]
 [![deps][deps]][deps-url]
-<a href="https://gatsbyjs.org/contributing/how-to-contribute/">
+<a href="https://webpack.js.org/contribute/">
```
Comment (evilebottnawi) [inline README.md]: Please use definitions

**Comment by harikrishnana2021** [comment]
cc @evilebottnawi

File: README.md [hunk 3]
```diff
@@ -9,7 +9,7 @@
 
 [![node][node]][node-url]
 [![deps][deps]][deps-url]
-[![tests][tests]][tests-url]
```
Comment (sokra) [inline README.md]: Please don't remove the tests badge.

File: README.md [hunk 4]
```diff
@@ -9,7 +9,7 @@
...
+[![PR's welcome][prs]][prs-url]
```
Comment (sokra) [inline README.md]: Re-place that. It's in-between all the tests badges

**Comment by harikrishnana2021** [comment]
cc @sokra

**Comment by sokra** [comment]
Thanks

**Pull request merged**